index.html

```diff
@@ -0,0 +1,316 @@
+                    <li><a href="#" class="footer-link">Live Events</a></li>
+                    <li><a href="#" class="footer-link">The Learning Network</a></li>
+                    <li><a href="#" class="footer-link">Tools & Services</a></li>
+                    <li><a href="#" class="footer-link">Multimedia</a></li>
+                    <li><a href="#" class="footer-link">Photography</a></li>
+                    <li><a href="#" class="footer-link">Video</a></li>
+                    <li><a href="#" class="footer-link">Newsletters</a></li>
+                    <li><a href="#" class="footer-link">TimeMachine</a></li>
+                    <li><a href="#" class="footer-link">NYT Store</a></li>
+                    <li><a href="#" class="footer-link">Times Journeys</a></li>
+                    <li><a href="#" class="footer-link">Manage My Account</a></li>
+                </ul>
+            </div>
+            <div>
+                <ul>
+                    <li>Subscribe</li>
+                    <li class="top-subscribe"><a href="#" class="footer-link">Home Delivery</a></li>
+                    <li class="top-subscribe"><a href="#" class="footer-link">Digital Subscriptions</a></li>
+                    <li class="top-subscribe"><a href="#" class="footer-link">Games</a></li>
+                    <li class="top-subscribe"><a href="#" class="footer-link">Cooking</a></li>
+                    <li class="mid-subscribe"><a href="#" class="footer-link">Email Newsletters</a></li>
+                    <li class="mid-subscribe"><a href="#" class="footer-link">Corporate Subscriptions</a></li>
+                    <li class="mid-subscribe mid-subscribe-last"><a href="#" class="footer-link">Education Rate</a></li>
+                    <li class="mid-subscribe"><a href="#" class="footer-link">Mobile Applications</a></li>
+                    <li class="mid-subscribe"><a href="#" class="footer-link">Replica Edition</a></li>
+                    <li class="mid-subscribe"><a href="#" class="footer-link">International</a></li>
+                    <li class="mid-subscribe"><a href="#" class="footer-link">Canada</a></li>
+                    <li class="mid-subscribe"><a href="#" class="footer-link">Español</a></li>
+                    <li class="mid-subscribe"><a href="#" class="footer-link">中文网</a></li>
+                </ul>
+            </div>
+        </div>
+    </footer>
+    <section>
+        <ul class="footer-ul">
+            <li>© 2021 The New York Times Company</li>
+            <li>NYTCo</li>
+            <li>Contact Us</li>
+            <li>Accessibility</li>
+            <li>Work with us</li>
+            <li>Advertise</li>
+            <li>T Brand Studio</li>
+            <li>Your Ad Choices</li>
+            <li>Privacy Policy</li>
+            <li>Terms of Service</li>
+            <li>Terms of Sale</li>
+            <li>Site Map</li>
+            <li>Help</li>
+            <li> Subscriptions</li>
+        </ul>
+    </section>
```

Comment:
### Optional Change
- [ ] Do you mind while conducting the above changes, implement the sticky section bottom of the footer that sticks to the bottom of the page as well?